roles/etcd/tasks/backup.force_new_cluster.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: backup/vars.yml
+
+- name: Move content of etcd backup under the etcd data directory
+  command: >
+    mv "{{ l_etcd_backup_dir }}/member" "{{ l_etcd_data_dir }}"
+
+- name: Set etcd group for the etcd data directory
+  command: >
+    chown -R etcd:etcd "{{ l_etcd_data_dir }}"
+
```

Comment:
should this be done only when not containerized?  At least with the system container, 'root' should own the directory (and probably with the Docker container it is the same)

Reply:
Just tested the migration with the containerized docker etcd, no failure.

Reply:
for the system container I needed to add: https://github.com/openshift/openshift-ansible/pull/5655/files#diff-15ef96fcf45a76727342f4da27343412R76 to avoid permission issues

Reply:
let's integrate the system containers integration later. I will be opening another PR, I can include it there as well.